IRremoteESP8266.cpp

```diff
@@ -856,7 +856,7 @@ long IRrecv::decodeNEC(decode_results *results) {
     results->decode_type = NEC;
     return DECODED;
   }
-  if (irparams.rawlen < 2 * NEC_BITS + 4) {
+  if (irparams.rawlen < 2 * NEC_BITS + 2 ) { // buf len is 64+(2)+1? and not 2*32+4
```

Comment:
I think the issue could be of how the lib and esp8266 works and require us for bigger tolerance.
I have some simple IR remote from some IR led kit which sometimes shows me unknown (but same) and sometimes NEC code. By changing the value of TOLERANCE from 25 to 35, I now always get NEC code instead of mixed unknown/nec all the time.

can you try if this fixes your issue?
if so we'll have to do more tests with more remotes to have a better value for tolerance.